rts/lra/lra-client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -185,6 +193,15 @@ private void init(String scheme, String host, int port) throws URISyntaxExceptio
             responseDataMap.clear();
     }
 
+    @Override
+    public void setRecoveryCoordinatorURI(URI uri) {
+        // TODO
```

Comment:
Is this intentional - if so add a JBTM?

Reply:
It was a personal reminder from a previous version so I will remove it

Reply:
I kept it and call out to setCoordinatorIURI instead (since they two coordinators are co-located in our implementation)